src/frontend/discussions-abusefilter-app/src/interfaces/LocalBackendInterface.ts

```diff
@@ -0,0 +1,38 @@
+import type { Filter, FilterMetadata } from '@shared/filters';
+import type { BackendInterface } from './BackendInterface';
+
+export class LocalBackendInterface implements BackendInterface {
+  filters: Filter[];
+  filtersMetadata: FilterMetadata[];
```

Comment:
Might be easier (for lookup) if this is a `Record<string, Filter>`, where string is the filter ID.